src/mono/System.Private.CoreLib/src/System/Environment.Mono.cs

```diff
@@ -21,6 +21,9 @@ public static extern int ExitCode
             set;
         }
 
+        // GetIsCpuQuotaLimited() is implemented in CoreCLR/NativeAOT
```

Comment:
Could either remove the comment, or maybe change to something like below:
```suggestion
        // This is not implemented for Mono
```